corpus/expressions.txt

```diff
@@ -812,8 +826,8 @@ MyIntDelegate a = (ref int i) => i + 1;
             (lambda_expression
               (parameter_list
                 (parameter
-                  (parameter_modifier)
-                  (predefined_type)
+                  (ref_type
+                    (predefined_type))
```

Comment:
I don't get why this changed. Seemingly it's at the same location in the parse tree, it's a parameter inside a lambda declaration. Why did the parsing order change?

Reply:
I think it's because previously it was seeing it as a parameter but now you've added the optional type there and then previously added ref types it matches the ref_type before it matches the parameter.

Reply:
Hmm, although the parameter part of the tree is still there so that's confusing.